Sources/Shared/Extensions/Array+Tailor.swift

```diff
@@ -24,7 +24,30 @@ public extension Array {
 
     return objects
   }
- 
+
+  /**
+   - Parameter name: String
+   - Returns: A mappable object array, otherwise it returns nil
+   */
+  func objects<T : SafeMappable>(name: String? = nil) throws -> [T] {
+    var objects = [T]()
+
+    if let name = name {
+      for dictionary in self {
+        guard let dictionary = dictionary as? JSONDictionary,
+          value = dictionary[name] as? JSONDictionary else { continue }
+        objects.append(try T(value))
+      }
+    } else {
+      for dictionary in self {
```

Comment:
Yes, but I mostly copy pasted already existing code. We can refactor it later on 😉